python/mypy.ini

```diff
@@ -0,0 +1,10 @@
+[mypy]
```

Comment:
not a blocker for this PR, but I'm curious how you came up with the rules here?  In particular:

1. I would not have expected us to need to ignore missing imports, and am not totally clear on what that means or what we are doing that causes it to be necessary,
2. I'm a little surprised that we didn't get any other type hint errors that would have required some of the other wonky rules to be configured in here... but that is excellent news if it's accurate!

Reply:
This seemed weird to me too, but searching the mypy docs said that this would safely suppress the `Skipping analyzing X: module is installed, but missing library stubs or py.typed marker` errors I was encountering. Those were the only errors being produced, so (surprisingly to me too) no further config was necessary. It sounds like we can add some metadata somewhere/somehow to remove the need for these directives, but I didn't want to go down that rabbit hole for this PR.

Reply:
gotcha, that sounds like the right call for now.